content/sensu-core/0.29/platforms/sensu-on-freebsd.md

```diff
@@ -115,9 +115,8 @@ connect to the configured [Sensu Transport][6].
    for your transport configuration. For more information, please visit the
    [transport definition specification][10]._
 
-2. Please refer to the configuration instructions for the corresponding
-   transport for configuration file examples (see [Redis][7], or [RabbitMQ][8]
-   reference documentation).
+2. If the transport being used is running on a different host, additional configuration is required to tell the sensu client how to connect to the transport.
+Please see [Redis][7] or [RabbitMQ][8] reference documentation for examples.
```

Comment:
For the Redis link, it might be better to link to `reference/redis/#configure-sensu` instead of `reference/redis/#sensu-redis-configuration`

Reply:
Done!